### PR TITLE
Test for speeding up CI

### PR DESCRIPTION
**Pull request by bhawanaprasain**
## Description
These tests help to speedup CI by sending requests in loops rather than in scenario outline.

## Motivation and Context

Sending HTTP requests to endpoints  using scenario outline takes considerable time .If we send these requests in loops time required to get status code minimizes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

### CODE REVIEW

**Comment by bhawanaprasain** [comment]
@individual-it @phil-davis please review on my PR.

**Comment by individual-it** [comment]
need to be on top of #35395
and should use as much as possible the steps created there

**Comment by phil-davis** [comment]
@bhawanaprasain this will be good to have. `apiAuthOCS` suite is one of the slowest, so it will be good to speed it up.
Please rebase and resolve conflicts.

File: tests/acceptance/features/apiAuthOcs/ocsDELETEAuth.feature [hunk 1]
```diff
@@ -2,29 +2,24 @@
...
-      | 1               |/cloud/users/user0                                   | 997      | 401       |
-      | 2               |/cloud/users/user0                                   | 997      | 401       |
-      | 1               |/cloud/users/user0/groups                            | 997      | 401       |
-      | 2               |/cloud/users/user0/groups                            | 997      | 401       |
-      | 1               |/cloud/users/user0/subadmins                         | 997      | 401       |
-      | 2               |/cloud/users/user0/subadmins                         | 997      | 401       |
+  Scenario: send DELETE requests to OCS endpoints as admin with wrong password
+    When the administrator requests these endpoints of specified API version with "DELETE" using password "invalid" then the status codes should be as listed
+      | ocs_api_version | endpoint                                             | ocs-code | http-code | group  |
+      | 1               | /apps/files_sharing/api/v1/remote_shares/pending/123 | 997      | 401       | group1 |
```
Comment (individual-it) [inline tests/acceptance/features/apiAuthOcs/ocsDELETEAuth.feature]: use the full path of the ocs endpoint here, then you can also reuse the steps you have done in the other PR

**Review by individual-it — CHANGES_REQUESTED** [review]

**Comment by bhawanaprasain** [comment]
@paurakhsharma @dpakach @skshetry please review on my PR.

File: tests/acceptance/features/bootstrap/OCSContext.php [hunk 2]
```diff
@@ -203,7 +218,7 @@ public function userSendsToOcsApiEndpointWithHeadersAndPassword(
 	/**
 	 * @When /^the administrator sends HTTP method "([^"]*)" to OCS API endpoint "([^"]*)" with headers using password "([^"]*)"$/
 	 *
-	 * @param string $verb
+	 * @param string $verb false, $body
```
Comment (paurakhsharma) [inline tests/acceptance/features/bootstrap/OCSContext.php]: This must be a copy-paste error, right?

File: tests/acceptance/features/bootstrap/OCSContext.php [hunk 3]
```diff
@@ -275,6 +290,29 @@ public function theAdministratorSendsHttpMethodToOcsApiWithBodyAndPassword(
 		);
 	}
 
+	/**
+	 * @When administrator sends request these endpoints with :method with body using password :password then the status codes should be as listed
```
Comment (paurakhsharma) [inline tests/acceptance/features/bootstrap/OCSContext.php]:
```suggestion
	 * @When administrator requests these endpoints with :method with body using password :password then the status codes should be as listed
```

**Review by paurakhsharma — COMMENTED** [review]
Some minor comments, please have a look

**Review by individual-it — APPROVED** [review]

**Comment by phil-davis** [comment]
Backport `stable10` #35586

**Pull request merged**